### ISSUE TITLE
Always use paths relative to the configuration file or rosetta-cli, not both

### ISSUE THREAD

**Issue by SamHellawell** [coinbase/rosetta-cli#215]
I have multiple rosetta configuration files, in separate directories (devnode, mainnet). I have a list of exempt accounts and a list of balances to bootstrap, aswell as specifying a constructor DSL file. I run rosetta-cli from the parent directory of `devnode` or `mainnet`. EG: path `/run/rosetta/devnode` I will be running rosetta-cli in `/run/rosetta`.

With that understood, my issue is quite simple:

My configuration is like:
```
  "exempt_accounts": "./devnode/exempt_accounts.json",
  "bootstrap_balances": "./bootstrap_balances.json",
```

`/devnode` contains `bootstrap_balances.json` and `exempt_accounts.json`

It does work like this, however you can see that it's odd that `exempt_accounts` required the `devnode` prefix and `bootstrap_balances` does not. Either it should be always relative to the config.json file or never. There are similar issues with `data_directory` (its relative to rosetta-cli invocation path). `constructor_dsl_file` is relative to the config.json file.

**Comment by septerr** [comment]
Hi @SamHellawell ! Thank you for raising this. We should definitely have a more consistent pattern for specifying paths. I am looking into this now. 🙏

**Comment by HyperPeek** [comment]
Fixed by PR #221, can be closed.
